cmd/daily-heal-ops.go

```diff
@@ -0,0 +1,89 @@
+/*
+ * Minio Cloud Storage, (C) 2019 Minio, Inc.
```

Comment:
```suggestion
 * MinIO Cloud Storage, (C) 2019 MinIO, Inc.
```

Please use this everywhere